mlkem/native/x86_64/x86_64_zetas.i

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2024 The mlkem-native project authors
+// SPDX-License-Identifier: Apache-2.0
+
+// WARNING: This file is auto-generated from scripts/autogenerate_files.py
+//          Do not modify it directly.
+
+// Table of zeta values used in the AVX2 NTTs
+// See autogenerate_files.py for details.
+
+31498, 31498, 31498, 31498, -758, -758, -758, -758, 0, 0, 0, 0, 0, 0, 0, 0,
+    14745, 14745, 14745, 14745, 14745, 14745, 14745, 14745, 14745, 14745, 14745,
```

Comment:
This alignment is broken somehow - `clang-format` does not seem to like this very much. I cannot think of a way to resolve that cleanly right now.

Reply:
Yes, me neither